<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="addService">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>731</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone No:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditName">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>113</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhone">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>113</width>
       <height>35</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSubmit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>370</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditNotes">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>104</width>
       <height>75</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>50</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>92</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
